.travis.yml

```diff
@@ -0,0 +1,35 @@
+# testing file for Travis
+# https://github.com/neuropoly/ivado-medical-imaging
+language: python
+python:
+  - "3.6"
+  - "3.7"
+matrix:
+  include:
+    - os: linux  # list of Linux env: https://docs.travis-ci.com/user/reference/overview/
+      name: "Ubuntu 18.04 (Bionic Beaver)"
+      dist: bionic
+    - os: linux
+      name: "Ubuntu 16.04 (Xenial)"
+      dist: xenial
+    - os: linux
+      name: "Ubuntu 14.04 (Trusty)"
+      dist: trusty
+    #- os: osx
+    #  name: "OSX 10.13 (High Sierra)"
+    #  osx_image: xcode9.4
+    #  language: shell
+
+before_install:
+  - python3 --version
+  - pip3 install -U pip
+  - pip3 install -U pytest
+  - pip3 install codecov
+
+install:
+  - pip3 install -e .
```

Comment:
the doc asks to use pip (not pip3)-- i would replicate what is in the doc (the purpose of this testing is to make sure that what a user is instructed to do will work in the long term)

Reply:
**pip** works on Linux but not on **OSX/ Windows** as it points to **Python 2.X**.
Any advice?

Reply:
did you look into these posts?
- https://github.com/travis-ci/travis-ci/issues/9744#issuecomment-426894873
- https://travis-ci.community/t/osx-default-environment-with-python-doesnt-work/6116/2

another easy solution would be to install python locally via distribs like minconda